src/components/navbar/buttons/hamburger.js

```diff
@@ -1,15 +1,59 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './hamburger.scss';
+import Structure from '../../../structure.json';
+import { IoIosArrowForward } from 'react-icons/io';
+import { IoIosArrowBack } from 'react-icons/io';
+import { GrFormSearch } from 'react-icons/gr';
+
+const Hamburger = ({ showLinks }) => {
+    const [menu] = useState(Structure['menu']);
+    const [showMenu, setShowMenu] = useState(showLinks);
+
+    const [subMenu] = useState(Structure['menu'][5]);
+    const [showSubMenu, setShowSubMenu] = useState(false);
+
+    const controlMenu = () => {
+        setShowMenu(!showMenu);
+        setShowSubMenu(!showSubMenu);
+    }
 
-const Hamburger = () => {
-    
     return (
         <div className='hamburger__links'>
-            <ul className='hamburger__links__ul'>
-                <li><a href="">home</a></li>
-                <li><a href="">shop</a></li>
-                <li><a href="">blog</a></li>
-                <li><a href="">contact</a></li>
+            <ul className={showMenu ? 'hamburger__links__ul hamburger__links__ul--menu' : 'hamburger__links__ul hamburger__links__ul--subMenu'}>
+                {showMenu &&
+                    menu.map((link) => {
```

Comment:
Is there a link in the menu, not a menu item?

Reply:
actually yes. i also used that just to differentiate main menu from submenu as they have different height values.